yarn.lock

```diff
@@ -14,7 +14,7 @@
   resolved "https://registry.yarnpkg.com/@ant-design/icons-svg/-/icons-svg-4.2.1.tgz#8630da8eb4471a4aabdaed7d1ff6a97dcb2cf05a"
   integrity sha512-EB0iwlKDGpG93hW8f85CTJTs4SvMX7tt5ceupvhALp1IF44SeUFOMhKUOYqpsoYWQKAOuTRDMqn75rEaKDp0Xw==
 
-"@ant-design/icons@^4.6.3", "@ant-design/icons@^4.7.0":
+"@ant-design/icons@^4.7.0":
```

Comment:
You deleted the lock file and generated it from scratch, right?

Reply:
Nope, I never delete the lock file, these are "natural" changes

Reply:
I guess there was an antd update yesterday that kicked in here

Reply:
Probably you're right and that's what triggered all the other dependency changes.

Reply:
Yes, yesterday: https://github.com/ant-design/ant-design/releases/tag/4.17.3